app/controllers/spree/users_controller.rb

```diff
@@ -48,12 +44,12 @@ def create
 
       if @user.save
         flash[:success] = t('devise.user_registrations.spree_user.signed_up_but_unconfirmed')
-        render cable_ready: cable_car.redirect_to(url: main_app.root_path)
+        redirect_to main_app.root_path
       else
-        render status: :unprocessable_entity, cable_ready: cable_car.morph(
-          "#signup-tab",
-          partial("layouts/signup_tab", locals: { signup_form_user: @user })
-        )
+        respond_to do |format|
+          format.html { head :unprocessable_entity }
+          format.turbo_stream { render :create, status: :unprocessable_entity }
+        end
```

Comment:
This change isn't covered by the controller spec. I _think_ it is covered by this spec: https://github.com/openfoodfoundation/openfoodnetwork/blob/master/spec/system/consumer/authentication_spec.rb#L90-L95

I can't verify right now, as there's something wrong with my dev environment 🫠 
Would you be able to double-check this? I think you should be able to see it if you run Chrome in non-headless mode like this:
```
HEADLESS=false bundle exec rspec spec/system/consumer/authentication_spec.rb:85
```

Reply:
That's correct, failed user signup spec is handled in `spec/system/consumer/authentication_spec.rb:85`